tools/Azure.Mcp.Tools.Extension/src/Commands/AzCommand.cs

```diff
@@ -180,7 +180,7 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
             await AuthenticateWithAzureCredentialsAsync(processService, _logger, cancellationToken);
 
             var azPath = FindAzCliPath() ?? throw new FileNotFoundException("Azure CLI executable not found in PATH or common installation locations. Please ensure Azure CLI is installed.");
-            var result = await processService.ExecuteAsync(azPath, command, _processTimeoutSeconds, cancellationToken: cancellationToken);
+            var result = await processService.ExecuteAsync(azPath, command, null, _processTimeoutSeconds, cancellationToken: cancellationToken);
```

Comment:
named arguments here too:

```suggestion
            var result = await processService.ExecuteAsync(azPath, command,
                operationTimeoutSeconds: _processTimeoutSeconds,
                cancellationToken: cancellationToken);
```